temporalcli/commandsmd/commands.md

```diff
@@ -57,6 +57,43 @@ This document has a specific structure used by a parser. Here are the rules:
 * `--color` (string-enum) - Set coloring. Options: always, never, auto. Default: auto.
 * `--no-json-shorthand-payloads` (bool) - Always all payloads as raw payloads even if they are JSON.
 
+### temporal activity: Complete or fail an activity.
+
+#### Options
+
+Includes options set for [client](#options-set-for-client).
+
+
+### temporal activity complete: Complete an activity.
+
+Complete an Activity Execution.
+
+`temporal activity complete --activity-id=MyActivityId --workflow-id=MyWorkflowId --result='{"MyResultKey": "MyResultVal"}'`
+
+#### Options
+
+* `--activity-id` (string) - The Activity to be completed. Required.
+* `--identity` (string) - Identity of operator.
+* `--result` (string) - The result with which to complete the Activity (JSON). Required.
```

Comment:
This works for me to meet parity. There are a few things we may want in the future to be as robust as `--input` for workflows:

* `--result` as a `string[]` to accept multiple results
* `--result-file` to accept files instead of strings
* `--result-meta` to support payload metadata
* `--result-base64` bool to support base64 binary strings

I wonder if we should create an issue here (same for `--detail` on failure)

Reply:
https://github.com/temporalio/cli/issues/446